src/fosslight_source/_scan_item.py

```diff
@@ -94,7 +94,7 @@ def get_row_to_print(self):
             print_rows = [self.file, self.oss_name, self.oss_version, ','.join(self.licenses), self.download_location, "",
                           ','.join(self.copyright),
                           "Exclude" if self.exclude else "",
-                          self.comment]
+                          self.comment, self.matched_lines, self.fileURL, self.vendor]
```

Comment:
In this way, the number of columns may be different for each row. 

The number of columns in the report can be unified by receiving the scanOSS inclusion or not as a parameter to the get_row_to_print function instead of dividing it by download_location and classifying it.

Reply:
added a parameter for get_row_to_print changed checking value to form a row.